packages/stores/src/queries/pools/types.ts

```diff
@@ -1,6 +1,14 @@
+import { ObservableQuery } from "@keplr-wallet/stores";
 import { WeightedPoolRaw, StablePoolRaw } from "@osmosis-labs/pools";
 import { Duration } from "dayjs/plugin/duration";
 import { CoinPretty } from "@keplr-wallet/unit";
+import { ObservableQueryPool } from "./pool";
+
+export interface IPoolGetter extends ObservableQuery {
```

Comment:
I'm not a fan of adding I to interfaces. What do you think if we rename this to `PoolGetter`?